index.html

```diff
@@ -24,14 +24,15 @@
 
         <link rel="icon" href="../assets/favicons/home.png" type="image/x-icon">
         <link rel="stylesheet" href="style.css">
+		<link rel="stylesheet" href="/assets/stylesheets/navbar.css">
 
         <meta name="viewport" content="width=device-width">
     </head>
     <body>
         <!--Navigation bar-->
         <ul>
             <li><a href="" class="home">Home</a></li>
-            <li><a href="/games">Games</a></li>
+            <li><a href="games/">Games</a></li>
             <li id="github"><a href="https://github.com/HEJOK254"><img src="../assets/logos/github-mark-white.svg" title="My Github" alt="GitHub logo"></a></li>
```

Comment:
Use / before `games` to specify that the url is from the root directory, potentially preventing bugs

i think

```suggestion
            <li><a href="/games/">Games</a></li>
```